plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/utils.ts

```diff
@@ -31,3 +32,12 @@ export const hasChildren = ( children ): boolean => {
 		return isObject( child ) && child.key !== null;
 	} );
 };
+
+/**
+ * Gets the total items description text from PHP-computed setting.
+ *
+ * @return {string} The description text for the total items.
+ */
+export const getTotalItemsDescription = (): string => {
```

Comment:
Also, a nit: in other places in code, we're referring to those as "totals item". Here, we're calling it "total items". Does it make sense to make it consistent and rename this to:


```suggestion
export const getTotalsItemDescription = (): string => {
```

Reply:
What do you think about just calling `getSetting` directly in the components instead of this helper (see previous reply to your comment where you questioned implementing in utils)?

Reply:
See [this comment](https://github.com/woocommerce/woocommerce/pull/61861#discussion_r2507162876).